packages/core/src/Stores/contract-replay-store.js

```diff
@@ -121,8 +121,6 @@ export default class ContractReplayStore extends BaseStore {
     }
 
     populateConfig(response) {
-        if (!this.switch_account_listener) return;
```

Comment:
I don't think we can delete something from existing store logic. Could you please elaborate on why we need this change? If it's needed we can expand functionality with passed flags

Reply:
This was causing an issue with populating the contract details. I was debugging this issue with Ahmad and found that this line is not relevant anymore. So, it's fine to remove this.

Reply:
Oh, I see, thank you for explanation)